server/e2e/internal_project_test.go

```
@@ -0,0 +1,15 @@
+
+import (
+	"testing"
+
+	"github.com/reearth/reearth-cms/server/internal/app"
+)
+
+// GET /assets/{assetId}
+func TestInternalGetProjectsAPI(t *testing.T) {
+	e := StartServer(t, &app.Config{}, true, baseSeeder)
+
+	e = e
```

golangci-lint (1.64.8)

assign: self-assignment of e to e

(govet)